pkg/services/chgm/chgm.go

```diff
@@ -207,7 +211,8 @@ func (c Client) investigateInstances() (InvestigateInstancesOutput, error) {
 	// fmt.Printf("stoppedInstances ::: %#v\n", stoppedInstances)
 
 	if len(stoppedInstances) == 0 {
-		return InvestigateInstancesOutput{UserAuthorized: true}, nil
+		// UserAuthorized: true so SRE will still be alerted for manual investigation
+		return InvestigateInstancesOutput{UserAuthorized: true, Error: "no non running instances found, terminated instances may have already expired"}, fmt.Errorf("no non running instances found, terminated instances may have already expired")
```

Comment:
If we return an error here, we will never create the note at PD. You have to do this instead:

```
return InvestigateInstancesOutput{UserAuthorized: true, Error: "no non running instances found, terminated instances may have already expired"}, nil
```